README.md

```diff
@@ -7,9 +7,57 @@
+and explains its uses and applications.
+
+### [Conclave Auction](./conclave-auction):
+This sample allows bidders to confidentially submit their bids to an enclave (A protected region of memory which cannot be accessed by the OS, Kernel or BIOS). The bids are processed confidentially in the enclave and the result is returned to concerned parties.
+
+### [Conclave Corda Trade](./conclave-corda-trade):
+This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes servers as clients which send encryped order from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
+
+### [Psi](./psi-sample):
+PSI problem refers to the problem of determining the common elements from the intersection of two sets without leaking or disclosing any additional information of the remaining elements of the either sets.
+Using this sample will see how Conclave (based on Intel SGX) can be a new tool to solve the private set intersection (PSI) problem.
+
+### [Tribuo Classification](./tribuo-classification):
+Using this sample will see how Conclave (based on Intel SGX) can be used in training a ML model. We will use tribuo Java ML library to load the AI model.
+Most of the hospitals/doctors have lot of their patients' data, which can be used to determine whether a tumor is malignant or begnin.
+Such data can be used to train an AI model. Once a model is trained, this model can be used to predict if a given tumor is malignant
+or begnin given certain input attributes.
+
+### [Tribuo Tutorials](./tribuo-tutorials):
+Tribuo is a Java machine learning library, which makes it well suited to run with Conclave. This sample provides tools for classification, regression, clustering, model development, and more.
+This sample shows you how you can use the [Tribuo Java ML](https://tribuo.org/learn/4.0/tutorials/) library to load, train many models like classification models, regression models, clustering models etc.
+
+### Initial SetUp
+
+In order to compile and run all the samples successfully, kindly download the conclave SDK
+and update the conclaveRepo property in GRADLE_USER_HOME/gradle.properties as shown below
+            
+    conclaveRepo=<path-to-conclave-sdk-repo>
+
+Below is a sample gradle.properties file, which shows you how to set conclaveRepo to point to the repo folder inside conclave-sdk.
+
+    cat gradle.properties
+    conclaveRepo=/Users/sneha.damle/Documents/projects/conclave/conclave-sdk-1.1/repo
+
+Please note, if GRADLE_USER_HOME is not set, it defaults to $USER_HOME/.gradle.
+Once set, Conclave will always pick up the conclaveRepo from gradle.properties. To override this behaviour you can pass your own custom conclave sdk path via command line as show below.
```

Comment:
May be helpful to link to the gradle docs here https://docs.gradle.org/current/userguide/build_environment.html